package.json

```diff
@@ -3,6 +3,10 @@
   "version": "1.1.0",
   "description": "Redact sensitive npm information from output",
   "main": "lib/index.js",
+  "exports": {
+    ".": "./lib/index.js",
+    "./server": "./lib/server.js"
+  },
```

Comment:
added exports to package.json +1 👍

Reply:
It's generally convention to expose `./package.json` as well, since the use of `exports` won't allow requiring it anymore.

Reply:
Can we add that here as well?